ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,35 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk, partitions=None, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :param partitions: Partitions DAL object list
+        :return: None
+        """
+        if partitions is None:
+            partitions = disk.partitions
+        if len(disk.partitions) == 0:
+            return
+
+        root_client = SSHClient(disk.storagerouter, username='root')
```

Comment:
Optimization possible here
You allow to pass partitions, but you don't check whether those partitions actually belong to the disk provided ==> Partitions not belonging to the disk could be umounted and wiped and error will be thrown saying that removing the partitions failed